### PR TITLE
Collector quick start

### PR DESCRIPTION
**Pull request by chalin**
This is a proposal to rename & slightly rework/refactor the Collector "getting started":

- Renames the Collector **Getting started** to **Quick start**, since that is more what the page actually is.
- Clarifies prerequisites
- Slightly reorders commands
- Offers some way to filter output, because otherwise it can be difficult to, e.g., find Span logs because the Collector output is very busy
- Renames the page appropriately and adds an alias to handle redirects

Thoughts @svrnm @theletterf et al.?

(For context, this is prep for #1635.)

---

**Preview & redirect test**:

- https://deploy-preview-3787--opentelemetry.netlify.app/docs/collector/quick-start/
- https://deploy-preview-3787--opentelemetry.netlify.app/docs/collector/getting-started

### CODE REVIEW

File: content/en/docs/collector/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+In this page you'll learn how to do the following in less than five minutes:
+
+- Set up and run the OpenTelemetry Collector.
+- Send telemetry and see it processed by the Collector.
+
+## Prerequisites
+
+Ensure that your developer environment has the following. This page assumes that
+you're using `bash` -- adapt configuration and commands as necessary for your
+preferred shell.
```
Comment (theletterf) [inline content/en/docs/collector/quick-start.md:27]:
```suggestion
Make sure that your developer environment has the following. This page assumes that
you're using `bash`. Adapt configuration and commands as necessary for your
preferred shell.
```

File: content/en/docs/collector/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+- [Go](https://go.dev/) 1.20 or higher
+- [`GOBIN` environment variable][gobin] is set; if not initialize it
```
Comment (theletterf) [inline content/en/docs/collector/quick-start.md:30]:
```suggestion
- [`GOBIN` environment variable][gobin] is set; if unset, initialize it
```

File: content/en/docs/collector/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  appropriately, for example[^1]:
+  ```sh
+  export GOBIN=${GOBIN:-$(go env GOPATH)/bin}
+  ```
+
+[^1]:
+    For more information, see
+    [Your first program](https://go.dev/doc/code#Command).
+
+## Set up
```
Comment (theletterf) [inline content/en/docs/collector/quick-start.md:40]:
```suggestion
## Set up the environment
```

File: content/en/docs/collector/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+   For an easier time seeing relevant output you can filter it:
+
+   ```sh
+   $GOBIN/telemetrygen traces --otlp-insecure \
+     --traces 3 2>&1 | grep -E 'start|traces|stop'
+   ```
+
+5. In the terminal window running the Collector container, you should see trace
+   ingest activity similar to what is shown below:
```
Comment (theletterf) [inline content/en/docs/collector/quick-start.md:91]:
```suggestion
   ingest activity similar to what is shown in the following example:
```

File: content/en/docs/collector/quick-start.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+       End time       : 2024-01-16 14:13:54.586 +0000 UTC
+       Status code    : Unset
+       Status message :
+   ...
+   ```
+
+6. Open <http://localhost:55679/debug/tracez> and select one of the samples in
+   the table to see the traces you've just generated.
+
+7. Once you are done, shutdown the Collector container, for example, using
```
Comment (theletterf) [inline content/en/docs/collector/quick-start.md:121]:
```suggestion
7. After you are done, shutdown the Collector container, for example, using
```

File: content/en/docs/collector/quick-start.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+<!-- markdownlint-disable ol-prefix blanks-around-fences -->
+
+The OpenTelemetry Collector receives [traces](/docs/concepts/signals/traces/),
+[metrics](/docs/concepts/signals/metrics/), and
+[logs](/docs/concepts/signals/logs/), processes the telemetry, and exports it to
+a wide variety of observability backends using its components. For a conceptual
+overview of the Collector, see [Collector](/docs/collector).
+
+In this page you'll learn how to do the following in less than five minutes:
```
Comment (theletterf) [inline content/en/docs/collector/quick-start.md:18]:
```suggestion
You are going to learn to do the following in less than five minutes: 
```

**Review by theletterf — CHANGES_REQUESTED** [review]
Nice! I like the changes. Sent some suggestions to reword things in line with our style recommendations.

**Comment by svrnm** [comment]
works for me, if @theletterf is happy with the changes, I am happy to merge it

**Comment by chalin** [comment]
Thanks all for the feedback, and for the careful review @theletterf. I've accepted all of your suggestions. Let me fix the formatting issues, etc. and then PTAL.

**Review by theletterf — APPROVED** [review]
Thanks so much! Approved!

**Comment by svrnm** [comment]
@open-telemetry/collector-approvers PTAL!

**Comment by svrnm** [comment]
@open-telemetry/collector-approvers PTAL!

**Review by mx-psi — APPROVED** [review]

**Pull request merged**